modules/vsphere-problem-detector-config-checks.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+[id="troubleshooting-vsphere-configuration_{context}"]
+= Configuration checks for the vSphere problem detector
+
+The following tables identify the configuration checks that are run by the detector.
+Some checks verify the configuration of the cluster.
+Other checks verify the configuration of each node in the cluster.
+
+.Cluster configuration checks
+[options="header",cols="20,50,^15,^15"]
+|===
+|Name
+|Description
+|Periodic
+|Installation
```

Comment:
Do some checks still run at installation as well?  If not, I can remove these two columns.  If yes, please review the checks that I wrote as running at installation or refer me to the data.